packages/mobile/src/screens/profile-screen/ProfileScreen.tsx

```diff
@@ -104,6 +106,15 @@ export const ProfileScreen = () => {
     }
   }, [status])
 
+  // Record screen view
+  useEffect(() => {
```

Comment:
just wondering why some are effect and some are focusEffect?

Reply:
I was using useEffect when the url is based on data inside the component and useFocusEffect when the url is a static value. Pushing this into the screen component means I can use useEffect across the board with a dependency on the url